proto-compiler/src/cmd/compile.rs

```diff
@@ -89,25 +89,55 @@ impl CompileCmd {
         // List available paths for dependencies
         let includes: Vec<PathBuf> = proto_includes_paths.iter().map(PathBuf::from).collect();
 
-        let attrs_serde = "#[derive(::serde::Serialize, ::serde::Deserialize)]";
+        let attrs_serde = r#"#[cfg_attr(feature = "std", derive(::serde::Serialize, ::serde::Deserialize))]"#;
         let attrs_jsonschema =
             r#"#[cfg_attr(feature = "json-schema", derive(::schemars::JsonSchema))]"#;
-        let attrs_serde_ord =
-            "#[derive(::serde::Serialize, ::serde::Deserialize, Eq, PartialOrd, Ord)]";
-        let attrs_serde_eq = "#[derive(::serde::Serialize, ::serde::Deserialize, Eq)]";
+        let attrs_ord =
+            "#[derive(Eq, PartialOrd, Ord)]";
+        let attrs_eq = "#[derive(Eq)]";
         let attrs_serde_default = "#[serde(default)]";
         let attrs_serde_base64 = r#"#[serde(with = "crate::base64")]"#;
         let attrs_jsonschema_str =
             r#"#[cfg_attr(feature = "json-schema", schemars(with = "String"))]"#;
 
         let compilation = tonic_build::configure()
             .build_client(true)
+            .compile_well_known_types(true)
```

Comment:
@Wizdave97 Do you need this for your use case or can we keep this change for a separate PR to make it easier to review?

Reply:
Yeah, the `Any` type is used in some of those structs so it also needs the serialize and deserialize implementation

Reply:
Can you please go into a bit more details? Is the issue that prost_types::Any does not implement (De)Serialize?

Reply:
Yeah prost_types::Any does not implement (De)Serialize and I felt making a longer time to get it merged

Reply:
Also seems like their release cycles are longer, the last release was October last year.